include/homestead_pd_definitions.h

```diff
@@ -106,10 +106,30 @@ static const PDLog2<const char*, int> CL_HOMESTEAD_HTTP_INIT_FAIL
   "(2). Check the network status and configuration. "
 );
 
-static const PDLog CL_HOMESTEAD_ENDED
+static const PDLog CL_HOMESTEAD_CACHE_INIT_FAIL
 (
   PDLogBase::CL_HOMESTEAD_ID + 9,
   LOG_ERR,
+  "Fatal - Failed to start the cache",
+  "The HSS cache could not be started.",
+  "The application will exit and restart until the problem is fixed.",
+  "Check shared and local configuration are correct."
+);
+
+static const PDLog CL_HOMESTEAD_NO_IMPU_STORE
+(
+  PDLogBase::CL_HOMESTEAD_ID + 10,
+  LOG_ERR,
+  "Fatal - No IMPU store specified",
+  "No IMPU store was specified in shared configuration.",
+  "The application will exit and restart until the problem is fixed.",
+  "Specify homestead_impu_store in shared config, and then re-upload shared config."
+);
+
+static const PDLog CL_HOMESTEAD_ENDED
+(
+  PDLogBase::CL_HOMESTEAD_ID + 11,
```

Comment:
I don't think so. I spoke to Rob and he agreed that he thought we could just change this